python/lsst/daf/butler/datastores/fileLikeDatastore.py

```diff
@@ -307,10 +325,13 @@ def addStoredItemInfo(self, refs, infos):
             )
         self._table.insert(*records)
 
-    def getStoredItemInfo(self, ref):
+    def getStoredItemInfo(self, ref: Union[DatasetRef, FakeDatasetRef]) -> StoredFileInfo:
```

Comment:
`DatasetIdRef` (in `registry.interfaces`) is a type alias for this union.

Open to ideas for better names for that.